### PR TITLE
Vendicated.Vencord.PTB version 1.2.9

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107244)

### CODE REVIEW

**Comment by solomoncyj** [comment]
wtf?

**Comment by solomoncyj** [comment]
Ok

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by solomoncyj** [comment]
https://github.com/microsoft/winget-cli/issues/1885

**Comment by solomoncyj** [comment]
.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 
2023-08-27 17:15:01.040 [CLI ] ShellExecute installer failed: 1

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 2023-08-28 17:14:29.289 [CLI ] Starting installer selection.  
> 2023-08-28 17:14:29.289 [CLI ] Installer [X86,exe,Unknown,] not applicable: Architecture was excluded by caller : X86  
> 2023-08-28 17:14:29.289 [CLI ] No suitable installer found for manifest Discord.Discord.PTB with version 1.0.1032  
> 2023-08-28 17:14:29.290 [CLI ] Terminating context: 0x8a150104 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DependenciesFlow.cpp:12b

File: manifests/v/Vencord/Vencord/PTB/1.2.10/Vencord.Vencord.PTB.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Vencord.Vencord.PTB
+PackageVersion: 1.2.10
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Discord.Discord.PTB
+Installers:
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/v/Vencord/Vencord/PTB/1.2.10/Vencord.Vencord.PTB.installer.yaml:10]:
```suggestion
- Architecture: x86
```
I'm not sure, but I believe the architecture has to be `x86` for the Discord call to work correctly.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Standard error:  Installation failed with exit code -1978335226 
2023-08-30 06:43:58.512 [CLI ] ShellExecute installer failed: 1 

I can't replicate this error in a Windows 10 VM.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**